README.md

```diff
@@ -1,17 +1,100 @@
 # Conda Recipes
 
-Example recipes for the new conda build system.  Use
+Community-maintained repository of conda package build recipes
+for 592+ conda packages.
 
-    conda build <recipe directory>
+* Package maintainers SHOULD upgrade packages
+  when appropriate.
 
-You have to use conda >= 1.7
-[conda](https://github.com/continuumio/conda).
+* Conda recipes MAY run other scripts:
```

Comment:
I see, but that's too much detail for a Readme, which IMHO should explain the big picture :-)